src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.html

```diff
@@ -1,19 +1,37 @@
-<form *ngIf="!isLoading; else loadingContainer" name="exportForm" role="form" (ngSubmit)="exportRepos(exercise.id)" #exportForm="ngForm">
+<form
+    *ngIf="!isLoading; else loadingContainer"
+    name="exportForm"
+    role="form"
+    (ngSubmit)="selectedProgrammingExercises.length <= 0 ? exportRepos(exercise.id) : bulkExportRepos()"
```

Comment:
Maybe we can introduce a new variable something like `isRepoExportForMultipleExercises = selectedProgrammingExercises.length > 0`